src/main/java/Duke.java

```diff
@@ -1,10 +1,17 @@
+import java.util.Scanner;
```

Comment:
In general the expressions are relatively simple. Also no magic numbers

Reply:
The code is also largely KISS compatible, easy for readers
SLAP is also implemented appropriately